modules/cpp-peglib/1.0.1/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  bazel:
+  - 8.x
+  - 7.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
```

Comment:
In general, adding `--process_headers_in_dependencies` as a build flag here would give you better test coverage. See https://github.com/search?q=repo%3Abazelbuild%2Fbazel-central-registry%20--process&type=code